app/config/ErrorHandler.scala

```diff
@@ -17,14 +17,28 @@
 package config
 
 import javax.inject.{Inject, Singleton}
+import models.ReturnToStartException
+import play.api.Logger
 import play.api.i18n.MessagesApi
-import play.api.mvc.Request
+import play.api.mvc.{Request, RequestHeader, Result, Results}
 import play.twirl.api.Html
 import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
 import views.html.error
 
 @Singleton
 class ErrorHandler @Inject()(val messagesApi: MessagesApi, errorTemplate: error) extends FrontendErrorHandler {
+
+  private val logger = Logger(classOf[ErrorHandler])
```

Comment:
`this.getClass()` will return the same and you will not have a risk that you'll change this class to something different, but this approach is correct too